mgmt/src/grpc/converter.rs

```diff
@@ -722,10 +780,20 @@ pub fn convert_bgp_neighbor_to_grpc(
         af_activate.push(2); // L2VPN_EVPN
```

Comment:
Does the rust autogenerated code for the schema have a symbol for 2 (L2VPN_EVPN)